clicache/src/IPdxReader.hpp

```diff
@@ -205,6 +206,11 @@ namespace Apache
           /// <param name="fieldName">The name of a member field.</param>
           /// <param name="type">The type of a member field, which value needs to read.</param>
           Object^ ReadField(String^ fieldName, Type^ type);
+
+          property Apache::Geode::Client::Cache^ Cache
```

Comment:
Should probably be `internal`.

Reply:
This is part of the public interface and we decided it can remain here after some discussion.